CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing
```

Comment:
Same as in the MOK, add a section in the main README linking this file.

Reply:
See https://github.com/KiweeEu/shopware-storefront-build-config/pull/3/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R20-R24